ci/scripts/run-unit-test.sh

```diff
@@ -4,6 +4,7 @@ set -euo pipefail
 echo "+++ Run unit tests with coverage"
 # use tee to disable progress bar
 NEXTEST_PROFILE=ci cargo llvm-cov nextest --lcov --output-path lcov.info --features failpoints,sync_point 2> >(tee);
+NEXTEST_PROFILE=ci RUSTFLAGS="--cfg loom" cargo nextest run --test loom
```

Comment:
How about making this a separate step by `echo "--- xx"`?

Reply:
+1

Reply:
I just add a line like above 😄